app/ui/browser-modern/constants/ui.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import * as ContentURLs from '../../../shared/constants/content-pages-locations';
+
+// Default location settings.
+export const HOME_PAGE = ContentURLs.HISTORY_PAGE;
+
+// Tabbar dimensions.
+export const TABBAR_HEIGHT = 40; // px
+export const TAB_DEFAULT_WIDTH = 15; // vw;
```

Comment:
I'm still torn between adding units to these consts or not. Having `15vw` instead of a plain `15` would mean having to do `parseInt`s when dynamically computing other dimensions based on these values.